src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string | null;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTabId }) => {
+  const selectedTab = tabs.find((tab) => tab.id === selectedTabId) || {
+    id: '',
+    content: '',
+  };
```

Comment:
let's not set anything if it wasn't found
```suggestion
  const selectedTab = tabs.find((tab) => tab.id === selectedTabId)
```